app/slackapp.js

```diff
@@ -243,20 +243,20 @@ function setupApp(slackapp, config, trustpilotApi) {
     const feedsMap = new Map(feeds.map((f) => [f.channelId, f]));
     feedsMap.delete(channelId);
     team.feeds = [...feedsMap.values()];
-    slackapp.saveTeamAsync = slackapp.saveTeamAsync || bluebird.promisify(slackapp.saveTeam);
+    slackapp.saveTeamAsync = slackapp.saveTeamAsync || promisify(slackapp.saveTeam);
     await slackapp.saveTeamAsync(team);
     showFeedSettingsIntroMessage(message, bot);
   };
 
   const handleSettingsCommand = async (bot, message) => {
-    bot.api.users.infoAsync = bot.api.users.infoAsync || bluebird.promisify(bot.api.users.info);
-      const { ok: userOk, user } = await bot.api.users.infoAsync({
-        user: message.user_id,
-      });
-      if (userOk && user.is_admin) {
+    bot.api.users.infoAsync = bot.api.users.infoAsync || promisify(bot.api.users.info);
+    const { ok: userOk, user } = await bot.api.users.infoAsync({
+      user: message.user_id,
+    });
+    if (userOk && user.is_admin) {
       await showFeedSettingsIntroMessage(message, bot);
-      } else {
-      bot.replyPrivateDelayedAsync = bot.replyPrivateDelayedAsync || bluebird.promisify(bot.replyPrivateDelayed);
+    } else {
+      bot.replyPrivateDelayedAsync = bot.replyPrivateDelayedAsync || promisify(bot.replyPrivateDelayed);
```

Comment:
Weird whitespace changes are due to interactive rebasing and changing the order of commits before pushing this. In the end, the linter saves us.